### ISSUE TITLE
About Code School QA Page

### ISSUE THREAD

**Issue by carmenkolohe** [carmenkolohe/CodeSchoolQA#3]
Create an about section.

It will describe the purpose of Code School QA, about everyone involved, twitch stream, the open source, how to collaborate and be a part of the community.

**Comment by dpkreativ** [comment]
HI @carmenkolohe I would like to work on this issue. Please let me know if it's available, and what I should do to contribute. Thanks

**Comment by jendevelops** [comment]
> Please create a directory in pages named about and a file in the directory called index.js
@carmenkolohe is it ok if I just create an `about.js` file instead? It doesn't look like there will be any other routes that utilize `/about/` route, so the directory is unnecessary.

**Comment by carmenkolohe** [comment]
@jendevelops Yes! That is a great idea! Please feel free to create an about.js. Yes, you are assigned :)
